pkg/api/v1/post_routes.go

```diff
@@ -33,7 +33,7 @@ func (a *API) handleReload(w http.ResponseWriter, r *http.Request) {
 
 	cfg, err := reloadConfig()
 	if err != nil {
-		logger.Error(err)
+		logger.Error(err.Error())
```

Comment:
Maybe the non-formatting directives could take a `Stringer` instead of a string? That way we could skip the explicit call here? Does that make sense?

Reply:
This one is tricky. If you find a way to do this obviously, would be really interesting to see.

Reply:
I think it makes sense. What I suggest to do then, is to do away with the convenience functions from the `slog` package and rolling our own also for `Info, Warning, Error and Debug`.

Reply:
Edit: it won't work, since `Info` and so on from `slog` already have variadic arguments in their function signature...